src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,17 @@
+%card-small-text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+%card-medium-text {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+%header-font {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
```

Comment:
In my opinion, it would be better use mixins for this

Reply:
Thanks, **fixed**